redbiom/_requests.py

```diff
@@ -16,12 +16,14 @@ def _format_request(context, command, other):
 
 def get_session():
     import redbiom
-    if redbiom.active_session is None:
-        import requests
-        redbiom.active_session = requests.Session()
-        redbiom.active_session.auth = redbiom.get_config()['auth']
+    import requests
+    import os
+
+    pid = os.getpid()
```

Comment:
note: this is the thread/multiprocess work around. What we do is create a session per process ID. These are automatically closed out `atexit`. This allows for each process to operate without mutating another's session state.